src/govuk/components/header/template.test.js

```diff
@@ -139,6 +139,20 @@ describe('header', () => {
       expect($navigationLink.html()).toContain('<em>Nav item</em>')
     })
 
+    it('renders navigation item with text without a link', () => {
+      const $ = render('header', examples['navigation item with text without link'])
+
+      const $navigationItem = $('.govuk-header__navigation-item')
+      expect($navigationItem.html()).toContain('Navigation item 1')
```

Comment:
I don't think this is testing quite what the test description says it's testing, because of the use of `.toContain()` – for example, even if the HTML looked like this:

```html
<li class="govuk-header__navigation-item">
    <a href="/super-sneaky-link">Navigation item 1</a>
</li>
```

then `$('.govuk-header__navigation-item').html()` would contain 'Navigation item 1' and the test would pass.

Reply:
Good spot 👍